custom_components/bosch_alarm/__init__.py

```diff
@@ -30,38 +32,30 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             host=entry.data[CONF_HOST], port=entry.data[CONF_PORT],
             automation_code=entry.data.get(CONF_PASSWORD, None),
             installer_or_user_code=entry.data.get(CONF_INSTALLER_CODE, entry.data.get(CONF_USER_CODE, None)))
-    try:
-        await panel.connect()
-    except asyncio.exceptions.TimeoutError:
-        _LOGGER.warning("Initial panel connection timed out...")
-    except:
-        logging.exception("Initial panel connection failed")
+
+    # The config flow sets the entries unique id to the serial number if available
+    # If the panel doesn't expose it's serial number, use the entry id as a unique id instead.
+    unique_id = entry.unique_id or entry.entry_id
+
+    # The config flow sets the entries title to the panel's model
+    model = entry.title.replace("Bosch ","")
```

Comment:
This is a kinda hacky... is there a way we can store the model explicitly alongside the user input in the entry?